queue/limited_serializable.go

```diff
@@ -388,13 +422,16 @@ func (q *limitedSizeSerializableLocal) SetRunner(r amboy.Runner) error {
 	return nil
 }
 
+// RetryHandler returns the queue's retry handler.
 func (q *limitedSizeSerializableLocal) RetryHandler() amboy.RetryHandler {
 	q.mu.RLock()
 	defer q.mu.RUnlock()
 
 	return q.retryHandler
 }
 
+// SetRetryHandler allows callers to inject a different retry handler
+// implementation if the queue has not yet started.
```

Comment:
`or if the retry handler has not yet been set.`

Reply:
It's allowed to overwrite the retry handler even if it's non-nil.

Reply:
But if it has started and the retry handler is non-nil, it can set it.

Reply:
I updated the comment.